common/trade_routes/00_wc_eastern_sea_kalimdor.txt

```diff
@@ -0,0 +1,40 @@
+eastern_kingdoms_sea_route = {  #Doesnt have trade posts since sea route
+	wealth = 10         # Total base wealth of route
+	trade_post_display_trigger = {
+		always = no
```

Comment:
If you set "always = no" in trade_post_display_trigger it doesn't mean you can't build it, it just means you can't see provinces where you can build it
Since there is no trigger for sea routes, you can build trade post in every sea route province, but you don't see that you can build them